src/modules/mavlink/mavlink_stream.cpp

```diff
@@ -66,6 +66,10 @@ MavlinkStream::update(const hrt_abstime &t)
 		// on the link scheduling
 		if (send(t)) {
 			_last_sent = hrt_absolute_time();
+
+			if (!_first_message_sent) {
```

Comment:
Why is _first_message_sent needed?

Reply:
To make sure that a first HL2 message is sent. I have to introduce an additional variable because _last_sent is not 0 anymore before a first message is sent.